research/README.md

```diff
@@ -10,17 +10,17 @@ Apache v2 License or equivalent open source license
 ## Requirements
 Each research project should create a sub directory with following requirements
 
-* sub directory name must be in ASCII string and no longer than 35 characters long
+* sub directory name must be in ASCII string, all in lower case and no longer than 35 characters long
```

Comment:
Shall we specify "snake_case"? In case some could use "kebab-case", or we allow both?

Reply:
sounds good